src/controllers/emailController.js

```diff
@@ -14,8 +13,16 @@ const sendEmail = async (req, res) => {
 
     console.log('to', to);
```

Comment:
console.log() should be removed or commented.

Reply:
thanks, done!